confgenerator/filter/internal/ast/ast.go

```diff
@@ -52,7 +52,7 @@ var logEntryRootStructMapToFluentBit = map[string]string{
 	"labels":         "logging.googleapis.com/labels",
 	"operation":      "logging.googleapis.com/operation",
 	"sourceLocation": "logging.googleapis.com/sourceLocation",
-	"httpRequest":    "logging.googleapis.com/http_request",
+	"httpRequest":    "logging.googleapis.com/httpRequest",
```

Comment:
Why not `confgenerator.HttpRequestKey` here as well?

Reply:
It results in a circular import; `confgenerator` imports `filter`, so `filter` can't import `confgenerator`. Not sure where a better spot for that constant would be than `confgenerator`.

Reply:
Yeah I think I agree with all of that. I added the explanatory comment.

A lot of projects have something like an `env` package for module-wide constants, unsure if that's the way we want to go or not.